redhat-developer/devspaces-images
# Fix linking of ~/.java/current to proper java binaries

On workspace startup it appeared `~/.java` directory is removed and then created again ( is is not exact that provided by universal developer image ), `~/.java/current` appeared to be missing and cannot be used to link the java binaries.

I've added as well a couple of `.gitignore` files to have more comfortable coding in VS Code startalone.

Solves https://issues.redhat.com/browse/CRW-4525

How to test
- create a workspace with https://github.com/vitaliy-guliy/demo-backend/tree/update-udi-to-checode
- open a terminal, check for java existence
- open `entrypoint.sh` from the root, check it contains similar with this PR files
